pkg/goo/install.ps1

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021, Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+Param(
+    [Parameter(Mandatory=$true)][string]$InstallDir
+)
+
+$ErrorActionPreference = 'Stop'
+
+$envFromMatch = {
+  Param($match)
+  (Get-ChildItem -Path Env: | `
+     Where-Object -Property Name -eq $match.Groups[1].Value).Value
+}
+$InstallDir = [regex]::Replace($InstallDir,'^<([^>]+)>',$envFromMatch)
```

Comment:
Do we *really* need this complicated machinery? Can you not get Googet to pass in an already-expanded path?

Reply:
If Googet supports using templates for this in the package definition, I think that's less distasteful. But maybe I don't understand what you mean.

Reply:
Looking at the Googet config again, I would guess that we can't do the variable substitution with Go templates because that would be baking the value of `PROGRAMDATA` from the build machine in, and not evaluating it at install time.

What if you just pass in a path relative to PROGRAMDATA and hardcode that environment variable as a prefix? That would significantly simplify the install script.